ui/src/components/features/build-panel/components/files-dialog.tsx

```diff
@@ -96,105 +171,117 @@ export default function FilesDialog({ form, classNames }: TFilesDialog) {
-              <CardContent className="p-6 space-y-4">
-                <div className="border-2 border-dashed border-gray-700 rounded-lg flex flex-col gap-1 p-6 items-center">
-                  <FileIcon className="w-12 h-12" />
-                  <span className="text-sm font-medium text-gray-500">
-                    Drag and drop a file or click to browse
-                  </span>
-                </div>
-                <div className="space-y-2 text-sm">
-                  <Label>File upload</Label>
-                  <Input
-                    placeholder="Picture"
-                    type="file"
-                    accept="image/*, application/pdf"
-                    onChange={(event) => {
-                      handleFileChange(event);
-                    }}
-                  />
-                </div>
-              </CardContent>
-              <CardFooter>
-                <Button
-                  variant="default"
-                  disabled={!fileUpload}
-                  size="lg"
-                  type="button"
-                  onClick={handleUpload}
-                >
-                  Upload file
-                </Button>
-              </CardFooter>
-            </Card>
-          </DialogDescription>
-        </DialogHeader>
-        <DialogFooter>
-          <DialogClose asChild>
-            <Button size="lg" type="button">
-              Close
+        <Form {...form}>
+          <form
+            onSubmit={form.handleSubmit(onSubmit)}
+          >
+            <DialogHeader>
+              <DialogTitle className="mb-3 text-slate-300">
+                Manage Files
+              </DialogTitle>
+            </DialogHeader>
+            <DialogDescription>
+              <FormField
+                control={form.control}
+                name="files"
+                render={({ field }) => {
+                  return (
+                    <FormItem className="flex flex-col">
+                      <FormControl>
+                        <MultiSelect
+                          values={values}
+                          placecholder="Select a file..."
+                          defaultValues={formattedAssistantFiles}
+                          onValueChange={(selections) => {
+                            const files = getFilesFromSelections(selections);
+                            field.onChange(files);
+                          }}
+                        />
+                      </FormControl>
+                    </FormItem>
+                  );
+                }}
+              />
+              <Card className="bg-slate-200">
+                <CardContent className="p-6 space-y-4">
+                  <div className="border-2 border-dashed border-gray-700 rounded-lg flex flex-col gap-1 p-6 items-center">
+                    <FileIcon className="w-12 h-12" />
+                    <span className="text-sm font-medium text-gray-500">
+                      Drag and drop a file or click to browse
```

Comment:
Drag and drop did not work for me. It was just opening a new browser tab with the file.

Reply:
Will not work - I'm just going to remove for now.